FirstDeliverable/CSV/csv.go

```diff
@@ -0,0 +1,67 @@
+package csv
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+)
+
+type Item struct {
```

Comment:
A best practice is:
Every *exported (PascalCase)* struct, function, interface, variable, constant, etc; should have a comment with a proper description:
```go
// Item - some description here
type Item struct {
```

Reply:
Apply this best practice in the whole code